### PR TITLE
Nadiia

### PR DESCRIPTION
**Pull request by Fleshka87**
First try

### CODE REVIEW

File: src/utils/MainUtils.java [hunk 1]
```diff
@@ -1,4 +1,7 @@
 package utils;
 
 public class MainUtils {
+    /*
+    hello team
```
Comment (emiryazicii) [inline src/utils/MainUtils.java:5]: Hey, Nadia. Thanks for joining us. If you already checked the previous pull requests , me and farkhod had already send 2 pull requests to make some changes in MainUtils class. We already have a conflict to resolve :) So after you try to make some changes in same class I believe you are also the part of the conflict now. No worries we will discuss about that on the next meeting which is on wednesday.

**Review by emiryazicii — APPROVED** [review]
Hey Nadia, as you see I and Farkhod had already push

File: src/utils/MainUtils.java (hunk 1 above)
Comment (Fleshka87) [reply src/utils/MainUtils.java:5]: We will figure this out. No problem at all

**Review by drnajimov — APPROVED** [review]
perfect

**Review by drnajimov — APPROVED** [review]

**Review by bataborisa — APPROVED** [review]
Your changes made this file great. Thank you for your commitment!

**Pull request merged**